internal/services/iotcentral/iotcentral_application_resource.go

```diff
@@ -73,12 +72,11 @@ func resourceIotCentralApplication() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeString,
 				Optional: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					string(iotcentral.F1),
-					string(iotcentral.S1),
-					string(iotcentral.ST1),
-					string(iotcentral.ST2),
+					string(apps.AppSkuSTOne),
+					string(apps.AppSkuSTTwo),
+					string(apps.AppSkuSTZero),
```

Comment:
F1 isn't present here?

Reply:
The new version of the SDK doesn't have F1 or S1 anymore. Even though the older SDK has F1 and S1 as valid enums the API will throw an error if you try and create this resource with F1 or S1.